tests/data/group_services/group_sssd/rule_sssd_offline_cred_expiration/comment.fail.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+
+SSSD_CONF="/etc/sssd/sssd.conf"
+
+dnf -y install sssd
```

Comment:
We don't want the test to work only on fedora, please change the logic to install package by appropriate packaging mechanism. You can get inspired by
`data/group_system/group_software/group_updating/group_updating_utils.sh`

Reply:
I will use `yum` as it should work on both, Fedora and RHEL. IMO this should be solved by the test suite which should export some variable for installing and removing packages based on the tested VM.